package-lock.json

```diff
@@ -20959,6 +20969,15 @@
 				"vue2-datepicker": "^3.6.3"
 			},
 			"dependencies": {
+				"@nextcloud/calendar-js": {
```

Comment:
Note: `@nextcloud/calendar-js 2.0` is still a dependency from `@nextcloud/vue`. I opened https://github.com/nextcloud/calendar-js/issues/380 to improve that.